must-gather/gather

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+TIMESTAMP=$(date +%Y%m%d_%H%M%S)
+OUTPUT_DIR="${OUTPUT_DIR:-must-gather/${TIMESTAMP}}"
+
+DEFAULT_NS=$(oc get ns -o name | grep "oot" | cut -d"/" -f 2)
+NS="${NS:-$DEFAULT_NS}"
```

Comment:
I think we could just do
```suggestion
NS="${NS:-oot-operator-system}"
```
It will be easier to find when we replace all `oot` references to `kmm`.

Reply:
Yes, this is what I did in the previous one, I will revert it.